libs/api/domains/signature-collection/src/lib/signatureCollection.resolver.ts

```diff
@@ -0,0 +1,199 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql'
+import { SignatureCollectionSuccess } from './models/success.model'
+import { SignatureCollectionService } from './signatureCollection.service'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  IdsAuthGuard,
+  IdsUserGuard,
+  CurrentUser,
+} from '@island.is/auth-nest-tools'
+import { UseGuards } from '@nestjs/common'
+import { SignatureCollection } from './models/collection.model'
+import { SignatureCollectionList } from './models/signatureList.model'
+import { SignatureCollectionIdInput } from './dto/id.input'
+import { SignatureCollectionSignature } from './models/signature.model'
+import { SignatureCollectionNationalIdsInput } from './dto/signatureListNationalIds.input'
+import { SignatureCollectionBulk } from './models/bulk.model'
+import { SignatureCollectionSignee } from './models/signee.model'
+import { SignatureCollectionListInput } from './dto/singatureList.input'
+import { SignatureCollectionFindSignatureInput } from './dto/findSignature.input'
+import { SignatureCollectionAreaInput } from './dto/area.input'
+
+@UseGuards(IdsAuthGuard, IdsUserGuard)
```

Comment:
You don't need both `IdsAuthGuard` and `IdsUserGuard`. `IdsUserGuard` is more strict as it extends the `IdsAuthGuard` but requires that a user `nationalId` is present in the token. I assume from the context that there is always authenticated users there so we should prefer the `IdsUserGuard`.

Then when you add logic to the service layer you have to figure out if we should apply scope check here, or if there is a microservice behind this that will do scope check for user authorization.